src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/Visitors/QueryGenerators/SqlQueryGenerator.cs

```diff
@@ -122,9 +122,10 @@ public object VisitSqlRoot(SqlRootExpression expression, SearchOptions context)
 
             if (!searchOptions.CountOnly)
             {
-                var sortOrder = searchOptions.GetFirstSortOrderForSupportedParam();
+                var (paramInfo, sortOrder) = searchOptions.GetFirstSortSupportedParam();
```

Comment:
paramInfo -> searchParamInfo
paramInfo was little confusing to me but may be its just me.